scripts/add_to_board.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+from github import Github
+from common import env
+
+
+IN_PROGRESS_COLUMN_ID = 6050136
+TEST_COLUMN_ID = 6050141
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--token", type=str, required=True)
+    parser.add_argument("--pull_request", type=int, required=True)
+    args = parser.parse_args()
+
+    repo = env("GITHUB_REPOSITORY")
+
+    g = Github(args.token)
+    repo = g.get_repo(repo)
+    pr = repo.get_pull(args.pull_request)
+
+    if pr.is_merged():
+        column_id = TEST_COLUMN_ID
+    else:
+        column_id = IN_PROGRESS_COLUMN_ID
```

Comment:
nit: you can also use ternary operator here

```suggestion
    column_id = TEST_COLUMN_ID if pr.is_merged() else IN_PROGRESS_COLUMN_ID
```